### PR TITLE
Cleanup some leftover changes in gpMgmt from error tables removal.

### PR DESCRIPTION
**Pull request by asubramanya**

### CODE REVIEW

File: gpMgmt/bin/gpload.py [hunk 1]
```diff
@@ -2332,12 +2325,9 @@ def create_staging_table(self):
...
     def count_errors(self):
         notice_processor(self)
-        if (self.error_table or self.log_errors) and not self.options.D and not self.reuse_tables:
+        if self.log_errors and not self.options.D and not self.reuse_tables:
             # make sure we only get errors for our own instance
-            if self.error_table:
-                queryStr = 'select count(*) from ' + self.error_table + " WHERE relname = '%s'" % self.extTableName
-            else:
-                queryStr = "select count(*) from gp_read_error_log('%s')" %self.extTableName
+            queryStr = "select count(*) from gp_read_error_log('%s')" %self.extTableName
```
Comment (hlinnaka) [inline gpMgmt/bin/gpload.py]: Not this patch's fault, but this needs escaping.

Comment (asubramanya) [inline gpMgmt/bin/gpload.py]: Yup. I will update this.

**Comment by lij55** [comment]
Looks good

**Comment by danielgustafsson** [comment]
LGTM

**Comment by asubramanya** [comment]
Thanks for the review. Pushed to master.

**Pull request merged**